# Lesson 2.数值型对象的创建

数值类型对象主要包括整数型（int）、浮点型（float）和复数型（complex）

### 1.创建整数型对象

In [1]:
a = 2

In [2]:
# 使用type函数查看对象类型
type(a)

int

int就是Python中的整数型对象。我们也可以用int函数来创建整数

In [15]:
int(3)

3

In [16]:
int(3.1)

3

In [17]:
int(3.4)

3

int函数的基本作用是将输入的数值取整后输出

### 2.创建浮点型对象  
简单理解，Python浮点型对象就是数学意义中带小数位数的对象

In [3]:
b = 3.14

In [4]:
b

3.14

In [5]:
type(b)

float

Python中用float指代浮点型对象，我们也可以使用同名函数创建浮点型对象

In [18]:
float(1)

1.0

In [6]:
# 使用科学计数法莱创建浮点型对象
c = 1.23e2

In [7]:
c

123.0

In [8]:
type(c)

float

在科学计数法中，e代表以10为底，e后面数字代表10的幂。  
例如：1.23e2代表1.23 * 100 = 123

**Point：**
- 通过科学计数法创建的数，即使是整数，也会用浮点型来表示；
- 浮点型也可用于存储整数数值。

In [9]:
f1 = 1

In [10]:
f2 = 1.0

### 3.创建复数对象

Python复数对象与数学意义中复数对象定义相同

In [12]:
# 创建一个实部为3、虚部为4的复数
a =  3 + 4j

In [13]:
a

(3+4j)

In [14]:
type(a)

complex